db/seeds.rb

```diff
@@ -60,34 +61,84 @@
   setting.value = ModelConfig.generation.default.last&.id
 end
 
-# DEV
+# PROVISIONING
 #
-if Rails.env == "development"
-  ModelServer.find_or_create_by!(name: "localhost") do |ms|
-    ms.url = model_endpoint
-    ms.provider = "ollama"
-  end
-
-  ModelConfig.find_or_create_by!(name: "mistral:instruct", model_server: ModelServer.first) do |mc|
-    mc.model = "mistral:instruct"
-    mc.temperature = 0.1
-  end
-
-  ModelConfig.find_or_create_by!(name: "gemma:7b", model_server: ModelServer.first) do |mc|
-    mc.model = "gemma:7b"
-    mc.temperature = 0.2
-  end
-
-  ModelConfig.find_or_create_by!(name: "all-minilm", model_server: ModelServer.first) do |mc|
-    mc.model = "all-minilm"
-    mc.embedding = true
-  end
+devel_model_servers = [
+  {
+    name: "localhost",
+    provider: "ollama",
+  }
+]
+
+devel_model_configs = [
+  {
+    name: "mistral:instruct",
+    model_server: "localhost",
+    model: "mistral:instruct",
+    temperature: 0.1,
+  },
+  {
+    name: "gemma:7b",
+    model_server: "localhost",
+    model: "gemma:7b",
+    temperature: 0.2,
+  },
+  {
+    name: "all-minilm",
+    model_server: "localhost",
+    model: "all-minilm",
+    embedding: true,
+  },
+  {
+    name: "nomic-embed-text",
+    model_server: "localhost",
+    model: "nomic-embed-text",
+    embedding: true,
+  }
+]
+
+provisioned_model_servers = JSON.parse(ENV.fetch("PROVISIONED_MODEL_SERVERS") {
+  Rails.env == "development" ? JSON.generate(devel_model_servers) : '[]'
+})
```

Comment:
To make it more readable and avoid converting a hash to JSON and then parsing it back into a hash, maybe something like this?

```suggestion
provisioned_model_servers = if ENV["PROVISIONED_MODEL_SERVERS"].present?
  JSON.parse(ENV["provisioned_model_servers"])
elsif Rails.env == "development"
  devel_model_servers
else
  []
})
```

Reply:
Oh right! I forgot to revisit those lol I knew you wouldn't like them